lego-webapp/pages/companies/+Page.tsx

```diff
@@ -198,7 +198,14 @@ const CompaniesPage = () => {
 
             <p className={styles.infoText}>
               Savner du en bedrift? Savner du noe informasjon om en bedrift? Ta
-              kontakt med Bedkom, vi tar gjerne imot innspill!
+              kontakt med{' '}
+              <a
+                className={styles.infoText}
```

Comment:
Does this need the `infoText` classname? Wouldn't the class on the parent paragraph cascade to this as well?